Content.Server/Sound/EmitSoundSystem.cs

```diff
@@ -34,27 +34,20 @@ public override void Initialize()
 
         private void PlaySound(BaseEmitSoundComponent component)
         {
-            if (!string.IsNullOrWhiteSpace(component._soundCollectionName))
+            if (!string.IsNullOrWhiteSpace(component.SoundCollectionName))
```

Comment:
does the sound collection even need to be nullable? i can't think of a legit situation where someone would serialize any of the emitsoundcomponents without a sound collection to actually play